quinn/src/connection.rs

```diff
@@ -127,6 +127,21 @@ where
                     .expect("spurious handshake data ready notification")
             })
     }
+
+    /// The local IP address which was used when the peer established
+    /// the connection
+    ///
+    /// This can be different from the address the endpoint is bound to, in case
+    /// the endpoint is bound to multiple interfaces.
```

Comment:
Would this be clearer without sacrificing correctness? I'm not sure it makes sense to be bound to multiple interfaces with a specific address.
```suggestion
    /// the endpoint is bound to a wildcard address like `0.0.0.0` or `::`.
```

Reply:
Yes, I think this is fine. Updated